framework/py/flwr/supernode/cli/flwr_clientapp.py

```diff
@@ -68,8 +67,7 @@ def _parse_args_run_flwr_clientapp() -> argparse.ArgumentParser:
     parser.add_argument(
         "--run-once",
         action="store_true",
-        help="When set, this process will start a single ClientApp for a pending "
-        "message. If there is no pending message, the process will exit.",
+        help="This flag is deprecated and will be removed in a future release.",
```

Comment:
```suggestion
        help="DEPRECATED. When set, this process will start a single ClientApp for "
        "a pending message. If there is no pending message, the process will exit.",
```

better to keep the original `help` message but with a clear `DEPRECATED` ?